client/jetpack-cloud/components/sidebar/style.scss

```diff
@@ -54,3 +58,13 @@
 		font-weight: 500;
 	}
 }
+
+.jetpack-cloud-sidebar__guided-tour {
+	&--is-large-screen {
```

Comment:
Can I ask what is the purpose of the `--is-large-screen` modifier here? Reading through the code, it appears we're never changing the class name. My hunch is that we could have `.jetpack-cloud-sidebar__guided-tour` be the container for `display: none;` as well as the `break-medium` breakpoint. This might also alleviate some confusion, if we're rendering `--is-large-screen` even on small screens.

Reply:
Initially, I thought we would make a separate tour for smaller screens. My idea was to have a class modifier to distinguish the tour for the smaller and large screens, but I guess this might confuse things. Maybe the right names were `-is-desktop` and `-is-mobile`.

I decided for now to remove the modifier and instead just have the styling within the `.jetpack-cloud-sidebar__guided-tour` container. We can always work on refactoring this once we have a separate tour for the mobile screens.